src/pages/allies.js

```diff
@@ -35,14 +35,8 @@ export default function Allies() {
           </Text>
         </Box>
         <AllyFilter onSearch={setAllyFilters} />
-        <AllyFeed filters={allyFilters} />
-        <Pagination
-          onPageChanged={pagination => {
-            // @TODO add pagination handler
-          }}
-          totalRecords={70}
-          pageLimit={5}
-        />
+        <AllyFeed />
```

Comment:
@ChrisBrownie55 ensure `filters={allyFilters}` is applied on the AllyFeed! This is the cause of the build failing here :)